src/plugins/remove/RemoveAction.js

```diff
@@ -93,7 +93,7 @@ export default class RemoveAction {
         }
 
         if (!this.isAlias(child, parent)) {
-            this.openmct.objects.mutate(child, 'location', null);
+            await this.openmct.objects.mutate(child, 'location', null);
```

Comment:
Pre-existing issue here. Composition should never be modified this way, this needs to use the composition API. We should fix this rather than building more tech debt on incorrect code.

Instead the code should be:
```
const composition = this.openmct.composition.get(parent);
composition.remove(child)
```

Reply:
I _think_ this will solve the issue that was being caused in the test... if not I'll let you know.